Fix and export margin-trim tests.

sammygill opened the pull request:
#### e52dfe829efd70d1b3bd5e7a325eca749d5b7aa3
<pre>
Fix and export margin-trim tests.
<a href="https://bugs.webkit.org/show_bug.cgi?id=250715">https://bugs.webkit.org/show_bug.cgi?id=250715</a>
rdar://104340028

Reviewed by Brent Fulgham.

Lots of small changes (changing title, renaming files, etc.) Feedback
provided in the WPT pull request:
<a href="https://github.com/web-platform-tests/wpt/pull/38019">https://github.com/web-platform-tests/wpt/pull/38019</a>

* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-end-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-end-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-end-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-end-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-start-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-start-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-start-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-start-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-non-adjoining-item-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-non-adjoining-item.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-end-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-end.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-start-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-start.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-block-multiline.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-grow-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-grow.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-inline-multiline-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-orthogonal-item-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-orthogonal-item.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-shrink-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-shrink.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-grow-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-grow.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-orthogonal-item-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-orthogonal-item.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-shrink-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-shrink.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-trim-all-margins-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-trim-all-margins.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-end-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-end.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-start-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-start.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-end-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-end.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-start-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-start.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-trim-ignores-collapsed-tracks-expected.html:

Canonical link: <a href="https://commits.webkit.org/259110@main">https://commits.webkit.org/259110@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ea1f335de25ddaa6c1b304eb2b9bec586b47f31c

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/103947 "Passed style check") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/77/builds/13062 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/36889 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/113177 "Built successfully") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/173474 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/14092 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/3958 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/96198 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/112268 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/109719 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/10865 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/93937 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/38575 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/92703 "Passed tests") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/25548 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/80226 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/6421 "Built successfully") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/70/builds/26929 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/82/builds/6580 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/84/builds/3461 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/12572 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/46459 "Passed tests") | | 
| [❌ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/6269 "Failed to push commit to Webkit repository") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/79/builds/8354 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

brentfulgham reviewed (approved):
LGTM, and has been reviewed by external CSS experts.

sammygill committed:
Fix and export margin-trim tests.
https://bugs.webkit.org/show_bug.cgi?id=250715
rdar://104340028

Reviewed by Brent Fulgham.

Lots of small changes (changing title, renaming files, etc.) Feedback
provided in the WPT pull request:
https://github.com/web-platform-tests/wpt/pull/38019

* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-end-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-end-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-end-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-end-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-start-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-start-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-start-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-block-start-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-non-adjoining-item-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-non-adjoining-item.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-end-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-end.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-start-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block-start.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-replaced-block.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-block-multiline.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-grow-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-grow.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-inline-multiline-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-orthogonal-item-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-orthogonal-item.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-shrink-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-column-shrink.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-grow-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-grow.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-orthogonal-item-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-orthogonal-item.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-shrink-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-row-shrink.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-trim-all-margins-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/flex-trim-all-margins.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-end-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-end.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-start-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block-start.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-block.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-end-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-end.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-start-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline-start.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-inline.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/grid-trim-ignores-collapsed-tracks-expected.html:

Canonical link: https://commits.webkit.org/259110@main
